target/linux/x86/64/config-default

```diff
@@ -201,7 +201,7 @@ CONFIG_INTEL_IDLE=y
 # CONFIG_IOMMU_DEBUG is not set
 # CONFIG_ISCSI_IBFT_FIND is not set
 # CONFIG_ITCO_VENDOR_SUPPORT is not set
-CONFIG_ITCO_WDT=y
+# CONFIG_ITCO_WDT is not set
```

Comment:
Why should this not be build into the kernel any more for this target?

Reply:
Because (1) it's not applicable to Pentium, Atom, or any flavor or AMD processor (k8-64 or k10), (2) because watchdogs are traditionally built as kmods, and (3) because you might want to configure the parameters that it's loaded with (without having to diddle the grub parameters).